File: Examples/WindowsDesktop/Components/ConnectRtdePopupBase.cs
using System.Collections.ObjectModel;
using System.Windows.Forms;

public partial class ConnectRtdePopupBase : Form
{

    public ConnectRtdePopupBase(string text)
    {
        InitializeComponent();
        this.Text = text;
    }



    protected virtual void tree_AfterCheck(object sender, TreeViewEventArgs e)
    {    }
}

File: UnderAutomation.UniversalRobots.Showcase.Forms.vb/Components/ConnectRtdePopupBase.vb
Imports System.Windows.Forms

Public Partial Class ConnectRtdePopupBase
    Inherits Form

    Public Sub New(text As String)
        InitializeComponent()
        Me.Text = text
    End Sub



    Protected Overridable Sub tree_AfterCheck(sender As Object, e As TreeViewEventArgs)
    End Sub
End Class
